pyqtgraph/graphicsItems/LegendItem.py

```diff
@@ -9,7 +9,12 @@
 from .GraphicsWidgetAnchor import GraphicsWidgetAnchor
 from .BarGraphItem import BarGraphItem
 
-__all__ = ['LegendItem']
+__all__ = ['LegendItem', 'LegendStyle']
+
+
+class LegendStyle:
```

Comment:
```suggestion
class LegendStyle(enum.IntEnum):
```

Reply:
If we do that, we'll have to hold off on merging this until after we deprecate py2 support (0.12 release?)

Reply:
Ah, didn't look closely enough to see that py2 was still supported, since it's been EOL almost a year.

Reply:
Don't get me started (maintaining CI pipelines with python2 and old Qt bindings has been a real challenge)...we're looking at adopting NEP-29, but first order of business is merge outstanding py2 compatible PRs, the contributors should have the benefit of (eventually) seeing their PRs be in a stable release supporting versions of python they could have been using at the time of submission...

Once the PR count is down to ~5ish, we'll likely attempt to resolve any bug regressions, do another release, and then kill off py2/py < 3.7 support.

Reply:
I also prefer enums more but have to accept python 2 here